### PR TITLE
Add proposal for Security Hub feature

### PR DESCRIPTION
**Pull request by stonezdj**
Add proposal for Security Hub feature

### CODE REVIEW

File: proposals/new/cve_search_report.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+1. As an administrator, he/she can check the total number of vulnerabilities scanned in current view (system/project/label), the report include the total number of vulnerabilities count by severity level: Critical, High, Medium, Low, None and Unknown. Most dangerous vulnerabilities, Most dangerous artifacts in current report.
+1. As an administrator, he/she can filter vulnerabilities by score range in the current scope, the scope should be project or system level.
+1. As an administrator, he/she can filter vulnerabilities by image tag in the current scope.
+1. As an administrator, he/she can filter vulnerabilities by package in the current scope.
+1. As an administrator, he/she can filter vulnerabilities by CVE id in the current scope.
+
+## Personas
+
+Only system administrators could access the Security Hub feature.
```
Comment (dioguerra) [inline proposals/new/cve_search_report.md:28]: Can we also think on adding a spetial role with reduced priviledges for a security auditing authority?

File: proposals/new/cve_search_report.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+| cve_id        | Search vulnerability information by CVE ID  |
+| severity        | Search vulnerability information by severity level  |
+| package        | Search vulnerability information by package name  |
+| cvss_v3_score        | Search vulnerability information by cvss v3 score  |
+
+3. Search Vulnerability information
+
+The Vulnerability is the security issue found in the artifact, it includes the package information and the CVE information, but it is not the CVE itself.
+```
+GET /api/v2.0/security/{scanner_id}/vul?q=xxx
```
Comment (dioguerra) [inline proposals/new/cve_search_report.md:177]: Can we add something like 'last time pulled? to the response data'

Also a query parameter to filter by last time pulled to show only  CVEs that affect images pulled in the last month

**Review by dioguerra — COMMENTED** [review]
This looks awesome!

**Review by wy65701436 — DISMISSED** [review]
lgtm

**Review by chlins — APPROVED** [review]
lgtm

File: proposals/new/cve_search_report.md (hunk 1 above)
Comment (stonezdj) [reply proposals/new/cve_search_report.md]: Could you please open a GitHub issue to track it?

File: proposals/new/cve_search_report.md (hunk 2 above)
Comment (stonezdj) [reply proposals/new/cve_search_report.md]: Because of the performance consideration, the last pull time of the artifact might be inaccurate or disabled.

**Review by zyyw — DISMISSED** [review]

**Review by wy65701436 — APPROVED** [review]
Lgtm

**Review by chlins — APPROVED** [review]
lgtm

**Review by zyyw — APPROVED** [review]

**Comment by OrlinVasilev** [comment]
@stonezdj maybe now we have to move this to completed proposals ?

**Pull request merged**